src/listenMqtt.js

```diff
@@ -0,0 +1,443 @@
+"use strict";
+var fbconnect = require("./mqtt/fbconnect");
+var utils = require("../utils");
+var log = require("npmlog");
+
+var identity = function () { };
+var lastSeqId = 0;
+
+var defaultFuncs = undefined;
```

Comment:
Why do this rather than just passing defaultFuncs into listenMqtt?

Reply:
Yeah sure I think that would be better.